colcon_gradle/task/gradle/build.py

```diff
@@ -141,6 +141,9 @@ async def _build(self, args, env):
         cmd += (args.gradle_args or [])
         cmd += ['--stacktrace']
 
+        # Add install_base to environment in GRADLE_INSTALL_PREFIX
+        env['GRADLE_INSTALL_PREFIX'] = args.install_base
```

Comment:
IIUC, this environment variable is not something Gradle typically expects. IMO, introducing this variable creates undesired coupling between Gradle projects and colcon.

Maybe there's a better way for colcon to determine what artifacts from the Gradle build to install? What we're currently doing (globbing jar files) seems brittle and limiting.

Reply:
Thanks for the reply.

I didn't find any Gradle's mechanism to do that. My solution is based on colcon behaviour when compiling a CMake project. colcon passes the installation directory to CMake using `CMAKE_INSTALL_PREFIX` and it is the CMake project which uses this directory to install things.

But I'm open to listen an alternative.